src/components/Upgrade/Navigation.js

```diff
@@ -59,27 +59,30 @@ Navigation.propTypes = {
   onNext: PropTypes.func.isRequired,
 }
 
-const NavButton = ({ type, ...props }) => (
-  <ButtonBase {...props}>
-    <div
-      css={`
-        display: flex;
-        align-items: center;
-        justify-content: center;
-        height: 53px;
-        width: 53px;
-        border-radius: 50%;
-        background: rgba(0, 0, 0, 0.03);
-        transform: rotate(${type === 'next' ? '0' : '180deg'});
-        &:active {
-          background: rgba(0, 0, 0, 0.1);
-        }
-      `}
-    >
-      <img src={arrowSvg} alt="" />
-    </div>
-  </ButtonBase>
-)
+const NavButton = ({ type, ...props }) => {
```

Comment:
```suggestion
function NavButton({ type, ...props }) {
```

Reply:
I'll convert the base `Navigation` component as well. :)